rbac/rbac/settings.py

```diff
@@ -394,31 +394,40 @@
     NOTIFICATIONS_TOPIC = None
 
 # Kafka settings
+KAFKA_SERVERS = []
+
 if KAFKA_ENABLED:
     KAFKA_AUTH = {}
     if ENVIRONMENT.bool("CLOWDER_ENABLED", default=False):
-        kafka_broker = LoadedConfig.kafka.brokers[0]
-        KAFKA_HOST = kafka_broker.hostname
-        KAFKA_PORT = kafka_broker.port
+        kafka_brokers = LoadedConfig.kafka.brokers
+        if not kafka_brokers:
+            raise ValueError("No kafka brokers available")
+        for broker in kafka_brokers:
+            kafka_host = broker.hostname
+            kafka_port = broker.port
+            kafka_info = f"{kafka_host}:{kafka_port}"
+            KAFKA_SERVERS.append(kafka_info)
+
         try:
-            if kafka_broker.authtype.value == "sasl":
+            if kafka_brokers.authtype.value == "sasl":
```

Comment:
`kafka_brokers` is list of brokers created few lines above (L402), I believe this list doesn't have attribute `authtype`

Reply:
Thank you for catching that!!

Reply:
there are more lines like that